app/assets/javascripts/oxalis/view/right-menu/mapping_info_view.js

```diff
@@ -80,6 +158,9 @@ function mapStateToProps(state: OxalisState): Props {
   return {
     position: getPosition(state.flycam),
     isMappingEnabled: state.temporaryConfiguration.isMappingEnabled,
+    mousePosition: state.temporaryConfiguration.mousePosition,
+    activeViewport: state.viewModeData.plane.activeViewport,
+    activeCellId: typeof state.tracing.activeCellId === "number" ? state.tracing.activeCellId : 0,
```

Comment:
The usual `state.tracing.activeCellId != null` check probably doesn't work, because of Flow?